services/headless-lms/models/src/exams.rs

```diff
@@ -259,7 +268,7 @@ pub async fn verify_exam_submission_can_be_made(
     let student_has_time =
         Utc::now() <= enrollment.started_at + Duration::minutes(exam.time_minutes.into());
     let exam_is_ongoing = exam.ends_at.map(|ea| Utc::now() < ea).unwrap_or_default();
-    Ok(student_has_time || exam_is_ongoing)
```

Comment:
Fix the problem when user starts the exam a couple of minutes before the exam ends, but will still be able to answer questions